static/js/submit-warning.js

```diff
@@ -0,0 +1,53 @@
+document.addEventListener('DOMContentLoaded', function () {
+  let form = document.querySelector('.form-context');
+  let submitFormButton = document.getElementById('submitForm');
+  let viewableCheckbox = document.getElementById('shareOnAutSPACEs');
+  
+  // Check that event listener is working
+  // if (viewableCheckbox) {
+  //   viewableCheckbox.addEventListener('change', function () {
+  //     console.log('viewableCheckbox checked:', viewableCheckbox.checked);
+  //   });
+  // }
```

Comment:
Could the commented code be removed, or could it still be useful for testing later?